.github/workflows/integration-tests.yml

```
@@ -43,6 +43,19 @@ jobs:
         run: |
           echo -e 'pcm.!default {\n type hw\n card 0\n}\n\nctl.!default {\n type hw\n card 0\n}' > ~/.asoundrc
 
-      - name: Run Synpress
-        run: yarn run test:e2e
-
+      - uses: docker://agilepathway/pull-request-label-checker:latest
+        id: check-if-tests-should-run
+        continue-on-error: true
+        with:
+          one_of: ready-for-tests, QA
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run Synpress with dashboard
+        if: ${{ needs.check-if-tests-should-run.outcome == 'success' || (github.event_name == 'push') }}
+        run: yarn run test:e2e record
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.DASHBOARD_KEY }}
```

I think it would be nice to call this the same way to avoid confusion. 
```suggestion
          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
```